x-pack/plugins/observability_solution/entity_manager/server/lib/entities/built_in/services.ts

```diff
@@ -10,6 +10,7 @@ import { BUILT_IN_ID_PREFIX } from './constants';
 
 export const builtInServicesEntityDefinition: EntityDefinition = entityDefinitionSchema.parse({
   id: `${BUILT_IN_ID_PREFIX}services`,
+  version: '1.0.0',
```

Comment:
nit: can we have this as a prerelease version number until it's finalized

Reply:
i.e. 0.whatever

Reply:
Fixed in https://github.com/elastic/kibana/pull/187692/commits/e459eb139380d5833201245a1b35e7ce92a9dbaa